influxql/iterator.gen.go

```diff
@@ -767,7 +767,7 @@ func (itr *floatInterruptIterator) Next() (*FloatPoint, error) {
 	if itr.count&0xFF == 0xFF {
 		select {
 		case <-itr.closing:
-			return nil, nil
+			return nil, itr.Close()
```

Comment:
I think the reason this returned nil was mostly because another part of the code is supposed to call `Close()` and I didn't want to double close. Is the query engine not calling close correctly? It's supposed to call `Next()` until it gets a `nil` and then call `Close()`.